crates/uv/tests/common/mod.rs

```diff
@@ -367,6 +367,21 @@ pub fn run_and_format<'a>(
     (snapshot, output)
 }
 
+/// Recursively copy a directory and its contents.
+pub fn copy_dir_all(src: impl AsRef<Path>, dst: impl AsRef<Path>) -> std::io::Result<()> {
+    std::fs::create_dir_all(&dst)?;
+    for entry in std::fs::read_dir(src)? {
+        let entry = entry?;
+        let ty = entry.file_type()?;
+        if ty.is_dir() {
+            copy_dir_all(entry.path(), dst.as_ref().join(entry.file_name()))?;
+        } else {
+            std::fs::copy(entry.path(), dst.as_ref().join(entry.file_name()))?;
```

Comment:
```suggestion
            fs_err::copy(entry.path(), dst.as_ref().join(entry.file_name()))?;
```

Reply:
Is this true in all cases? Should we set up https://rust-lang.github.io/rust-clippy/master/index.html#/disallowed_methods

Reply:
There are some cases where we need `std::fs` (e.g. there's no equivalent to `std::fs::Permissions`), otherwise i'd always use `fs_err`, it helps tremendously with debugging.

Reply:
It sounds like we could use `disallowed_methods` for the methods that are e.g. not `fs::Permission`

Reply:
That'd be nice to have

Reply:
https://github.com/astral-sh/uv/issues/1916